test/utils/matchers.go

```diff
@@ -198,16 +198,25 @@ func (matcher *exitCleanlyMatcher) Match(actual interface{}) (success bool, err
 		return false, nil
 	}
 
-	// FIXME: #19809, "failed to connect to syslog" warnings on f38
-	// FIXME: so, until that is fixed, don't check stderr if containerized
-	if !Containerized() {
-		info := GetHostDistributionInfo()
-		if info.Distribution == "fedora" {
-			if stderr != "" {
-				matcher.msg = fmt.Sprintf("Unexpected warnings seen on stderr: %q", stderr)
-				return false, nil
-			}
-		}
+	// Exit status is 0. Now check for anything on stderr... except:
```

Comment:
Since I was on PTO until now: Very nice work! Not perfect but more than good enough for now to at least catch most of the weird errors.